pyglet/window/cocoa/__init__.py

```diff
@@ -395,7 +395,7 @@ def set_location(self, x, y):
         origin = cocoapy.NSPoint(x, screen_height - y - rect.size.height)
         self._nswindow.setFrameOrigin_(origin)
 
-    def get_size(self):
+    def get_size(self) -> Tuple[int, int]:
```

Comment:
I'm not sure about the intended functionality of `get_size`. I see two options:
1) Return the self._width and self._height as we are doing in the other classes
2) Return the size of the window as reported by each os specific library
#1 is simpler and platform agnostic, #2 is in theory more accurate

Reply:
I believe 1) would be the way to go. We have a more recent `get_framebuffer_size` method, which will return the _actual_ size if applicable.

Reply:
So it's safe to remove this function like I did for the others?

Reply:
Yes I believe it should be, but we will want to have someone with a Mac confirm it. I think we can merge it for now, and then revert later after testing.

Reply:
Ok sounds good